src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import PropTypes from 'prop-types';
+
+const Button = (props) => {
+  const clickHandler = () => props.clickHandler(props.name);
+  
+  return (
+    <button type="button" className="button" style={{ backgroundColor: props.color, width: (props.width === true ? '50%' : '') }} onClick={() => clickHandler()}>
```

Comment:
it's look neater if you lift out the styling and just pass a variable to style:
```
const style = {
background: this.props.colour
}
```
and just give the style variable to style on the button
[OPTIONAL]